include/modem/nrf_modem_lib.h

```diff
@@ -38,27 +38,37 @@ enum nrf_modem_mode {
 };
 
 /**
- * @brief Initialize the Modem library and turn on the modem.
+ * @brief Initialize the Modem library in normal operating mode and turn on the modem.
  *
- * This function initializes all integration components of the nrf_modem library
- * and turns on the modem. The operation can take a few minutes when a firmware update
- * is scheduled. If your application supports modem firmware updates, consider initializing
- * the library manually to have control of what the application should do in the meantime.
+ * This function initializes the modem in normal operating mode and initializes all integration
+ * components of the Modem library and turns on the modem. The operation can take a few minutes
+ * when a firmware update is scheduled.
```

Comment:
Some repetitions here:
`this function initializes the modem ... and turns on the modem`, those are the same thing.

Reply:
I think the first part can be omitted, and we could start with `The operation can take a few ...`